### PR TITLE
docs: use CodeExampleTabs from card to combobox

### PR DESCRIPTION
**Pull request by iropolo**
# docs: use CodeExampleTabs from card to combobox

Jira ticket: https://dialpad.atlassian.net/browse/DLT-1573

Implemented our new CodeExampleTabs component into:

- card
- checkbox
- checkbox_group
- chip
- codeblock
- collapsible
- combobox

### CODE REVIEW

File: apps/dialtone-documentation/docs/components/card.md [hunk 1]
```diff
@@ -82,7 +83,21 @@ They should be easy to scan for relevant and actionable information. Elements, l
...
+  <template #header>
+    (header slot)
+  </template>
+  <template #content>
+    (content slot)
+  </template>
+  <template #footer>
+    (footer slot)
+  </template>
+</dt-card>
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/card.md:98]: This coloring was fixed in the other PR right? probably just have to merge in staging.

Screenshot 2024-03-07 at 1 09 56 PM

File: apps/dialtone-documentation/docs/components/checkbox-group.md [hunk 2]
```diff
@@ -50,18 +37,86 @@ storybook: https://dialtone.dialpad.com/vue/?path=/story/components-checkbox-gro
   </dt-checkbox-group>
 </code-well-header>
 
-```html
+<code-example-tabs
+htmlCode='
+<fieldset
+  class="d-input-group__fieldset d-input-group__fieldset"
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/checkbox-group.md]: duplicate d-input-group__fieldset

File: apps/dialtone-documentation/docs/components/checkbox-group.md [hunk 3]
```diff
@@ -50,18 +37,86 @@ storybook: https://dialtone.dialpad.com/vue/?path=/story/components-checkbox-gro
...
+          <input
+            type="checkbox"
+            name="fruits-checkbox-group"
+            class="d-checkbox"
+            value="apple"
+          >
+        </div>
+        <div class="d-checkbox__copy d-checkbox__label">
+          <div class="">
+            <span>Apple</span></div>
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/checkbox-group.md]: `</div>` on next line

File: apps/dialtone-documentation/docs/components/card.md (hunk 1 above)
Comment (iropolo) [reply apps/dialtone-documentation/docs/components/card.md:98]: Yes, looks good now:
image

**Review by francisrupert — APPROVED** [review]
Looking good!

**Pull request merged**